database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -16,9 +16,13 @@ public function up()
         Schema::create('users', function (Blueprint $table) {
             $table->id();
             $table->string('name');
+            $table->string('phone');
             $table->string('email')->unique();
             $table->timestamp('email_verified_at')->nullable();
             $table->string('password');
+            $table->string('role');
```

Comment:
use default role 'user'. And don't use 'user' as a string here. Declare the Roles in the User Model as constants and use the constant here.

Reply:
Don't no how to use it. google it but not found.